blog/releases/announcing-knative-v0-22-release.md

```diff
@@ -0,0 +1,195 @@
+- `kn` [CLI v0.22](#client-v022)
+    - `kn` [CLI Plugins](#cli-plugins)
+- [Operator v0.22](#operator-v022)
+- [Thank you contributors](#thank-you-contributors)
+
+
+
+### Highlights
+
+- Serving Domain Mapping improves multi-tenency support to avoid domain name 'squatting' in cluster.
+- Eventing now allows Trigger users to set another namespace's Subscriber
+- Eventing kafka broker now Kubernetes's minimum version is 1.18
+- Eventing kafka now supports the ability to choose between ordered and unordered delivery
+- The CLI `kn` 0.22.0 comes with some bug fixes and minor feature enhancements. It's mostly a polishing release. If you are using the client API, there is a breaking change that was needed to align with Kubernetes client API's
+- There are two new CLI plugins align with 0.22 release, [kn-plugin-admin](https://github.com/knative-sandbox/kn-plugin-admin) and [kn-plugin-source-kafka](https://github.com/knative-sandbox/kn-plugin-source-kafka)
+- The Knative Operator release with some bug fixes, and supports the latest version of knative serving and eventing.
+
+
+### Serving v0.22
+
+#### 💫 New Features & Changes
+
+- Added an autoscaling annotation to choose a different aggregation algorithm for the autoscaling metrics. This is experimental currently. [#10840](https://github.com/knative/serving/pull/10840)
+- Added autocreateClusterDomainClaims flag to network config map. [networking/#330](https://github.com/knative/networking/pull/330)
+
+#### 🐞 Bug Fixes
+
+- Adds validation that a default max-scale is set if a max-scale-limit is specified in the autoscaler configmap (since otherwise the default max-scale, i.e. 0 = no max, would fail validation as it is above the max-scale-limit). [#10921](https://github.com/knative/serving/pull/10921)
+- Bumped the resource request and limits of the autoscaler to 100m/100Mi, 1000m/1000Mi respectively. [#10865](https://github.com/knative/serving/pull/10865)
+- Fixed a regression where the pod bringup time might have a latency of 10s or more even though the container should be up quickly. [#10992](https://github.com/knative/serving/pull/10992)
+- Reduced the necessary memory allocations in the activator significantly, especially with disabled tracing. [#11016](https://github.com/knative/serving/pull/11016), [#11013](https://github.com/knative/serving/pull/11013), [#11009](https://github.com/knative/serving/pull/11009), [#11008](https://github.com/knative/serving/pull/11008)
+- Fix the incorrect Gateway name format for DomainMapping auto TLS feature for net-istio implmenetation. [net-istio#532](https://github.com/knative-sandbox/net-istio/pull/532)
+
+
+### Eventing v0.22
+
+#### 🚨 API Changes
+
+- v1alpha1 Channel duck types are no longer supported. [#5005](https://github.com/knative/eventing/pull/5005)
+- Allow Trigger users to set another namespace's Subscriber, And when Subscriber's namespace not been set, it will be set to Trigger's namespace [#4995](https://github.com/knative/eventing/pull/4995)
```

Comment:
Again, it's not clear to me what you're trying to say here re "set another namespace's Subscriber".

From a docs perspective;
- Please use lowercase for triggers, subscribers
- Do no use possessive ('s) for namespace and other inanimate objects. E.g. "If the namespace for a subscriber has not been set, the subscriber defaults to using the namespace of the trigger."

Reply:
I really don't know what is this trying to say either is not clear to me.
I copy from the release notes here 
https://github.com/knative/eventing/releases/tag/v0.22.1

![image](https://user-images.githubusercontent.com/1094878/115813326-f9819700-a3c0-11eb-8327-8d607e7f86b5.png)

I think what it means is that now Triggers allow specifying a Subscriber in a different namespace, when the 
namespace of the Subscriber is not specified it will default to the same namespace as the Trigger.

@n3wscott is my description accurately above ^^ ?

I will comment it out for now, but I didn't see the docs updated to cover this new enhancement

Reply:
Yes that's correct. Now in the trigger subscriber you can specify the namespace of the subscriber, if it's not specified then it'll default to the same namespace of the trigger.

Reply:
@devguyio thanks for clarifying! If there's no docs issue to add this change could you open one please?